internal/backend/runtime/omni/controllers/omni/machine_status_test.go

```diff
@@ -429,7 +454,7 @@ func (suite *MachineStatusSuite) TestMachineSchematic() {
 			},
 			expected: &specs.MachineStatusSpec_Schematic{
 				Id:               defaultSchematic,
-				InitialSchematic: defaultSchematic,
+				InitialSchematic: "",
```

Comment:
I think this was a bug - if the machine is in agent mode, it means it is completely reset, we cannot talk about its initial state anymore. So now, we reset the initial state (the schematic id and the kernel args/extensions) to be empty.

The initial state was incorrect anyway, as we explicitly do not return any extensions when the machine was in agent mode.